tesseract_motion_planners/include/tesseract_motion_planners/descartes/profile/descartes_default_plan_profile.h

```diff
@@ -63,20 +63,25 @@ class DescartesDefaultPlanProfile : public DescartesPlanProfile<FloatType>
 
 #ifndef SWIG
   DescartesEdgeEvaluatorAllocatorFn<FloatType> edge_evaluator{ nullptr };
+  DescartesStateEvaluatorAllocatorFn<FloatType> state_evaluator{ nullptr };
 
   // If not provided it adds a joint limit is valid function
   DescartesVertexEvaluatorAllocatorFn<FloatType> vertex_evaluator{ nullptr };
 #endif
 
+  bool allow_collision{ false };
+
   // Applied to sampled states
   bool enable_collision{ true };
   tesseract_collision::CollisionCheckConfig vertex_collision_check_config{ 0 };
 
   // Applied during edge evaluation
   bool enable_edge_collision{ false };
   tesseract_collision::CollisionCheckConfig edge_collision_check_config{ 0 };
+
+  bool use_redundant_joint_solutions{ false };
```

Comment:
Doxygen. Also, this is a behavior change, because before it would use all redundant solutions? Should the default be true?

Reply:
That's probably a question of personal preference. To me it seems like more of a niche option that you would enable for special planning circumstances. When it is enabled, it can generate a huge amount of extra edges, adding to the existing problem of solving time and memory consumption.

Reply:
I am good with the default being false. It would be good to add Doxygen explaining the behavior if true versus false.

Reply:
Sounds good. I'll add the comments